install.sh

```diff
@@ -85,69 +85,51 @@ fi
 # Force curl to use system libraries.
 function curl () {
   # Retry if download failed.
-  # The --retry/--retry-all-errors parameter in curl will not work with 
+  # The --retry/--retry-all-errors parameter in curl will not work with
   # the 'curl: (7) Couldn't connect to server' error, a for loop is used
   # here.
   for (( i = 0; i < 4; i++ )); do
-    env LD_LIBRARY_PATH='' ${CURL} --ssl -C - "${@}" && \
-      return 0 || \
-      echo_info "Retrying, $((3-$i)) retries left."
+    env LD_LIBRARY_PATH=''
```

Comment:
`env` will not have any effect when not followed by a command

Reply:
Do we even need the `env` in front of the `LD_LIBRARY_PATH` assignment there? POSIX `sh` variable assignment isn't my strong suit.

Reply:
Not necessarily, it should work without `env` (however `env -u LD_LIBRARY_PATH <cmd>` is more self-explained (unsetting `LD_LIBRARY_PATH`)